official/resnet/keras/keras_imagenet_main.py

```diff
@@ -28,6 +28,7 @@
 from official.utils.flags import core as flags_core
 from official.utils.logs import logger
 from official.utils.misc import distribution_utils
+from tensorflow.python.keras import backend
```

Comment:
is this needed? we could just do `tf.keras.backend.set_...` ?

Reply:
Done.